### PR TITLE
view in Settings

### PR DESCRIPTION
**Pull request by testappN9**

### CODE REVIEW

**Comment by testappN9** [comment]
https://user-images.githubusercontent.com/92698997/155518243-55d73359-d5ef-4cea-b6a6-c95415c81ad0.mp4

**Comment by testappN9** [comment]
https://user-images.githubusercontent.com/92698997/155705256-28f0cf2f-6259-4b86-8397-c53ea3235047.mp4

File: tg/CustomSegmentedControl/CustomSegmentedControlView.swift [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+import UIKit
+
+class CustomSegmentedControl: UIControl {
+    
+    private let viewForSections = UIView()
+    private let backgroundView = UIView()
+    private let selector = UIView()
+    private var selectedIndex = 1
+    private var arrayOfSections: [String] = []
+    private var numberOfSections = 2
```
Comment (galinasmolskaya) [inline tg/CustomSegmentedControl/CustomSegmentedControlView.swift]: ???

File: tg/Settings/View/SettingsViewController.swift [hunk 2]
```diff
@@ -28,6 +32,23 @@ class SettingsViewController: UIViewController, SettingsPresenterDelegate {
...
     }
+    
+    func addingCustomSegmentedControl() {
+        viewForSegmentedControl.layoutIfNeeded()
+        customSegmentedControl = CustomSegmentedControl(frame: viewForSegmentedControl.bounds)
+        customSegmentedControl?.addSection(label: "First!")
+        customSegmentedControl?.addSection(label: "Second")
+        customSegmentedControl?.addSection(label: "Third")
+        customSegmentedControl?.addSection(label: "Fourth")
+        customSegmentedControl?.font(size: 20)
```
Comment (galinasmolskaya) [inline tg/Settings/View/SettingsViewController.swift]: ?????

**Review by galinasmolskaya — APPROVED** [review]

File: tg/CustomSegmentedControl/CustomSegmentedControlView.swift [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    func getSelectedIndex() -> Int {
+        return selectedIndex
+    }
+    
+    func setFont(size: CGFloat) {
+        sizeOfFont = size
+    }
+    
+    func setSections(labels: [String]) {
+        arrayOfSections += labels
```
Comment (galinasmolskaya) [inline tg/CustomSegmentedControl/CustomSegmentedControlView.swift:90]: ?

**Pull request merged**